src/components/ProfileForm/components/ProfileFormStep0.tsx

```diff
@@ -0,0 +1,59 @@
+const ProfileFormStep0 = () => {
+  return (
+    <>
+      <form className="m-1 p-1 mb-4">
```

Comment:
The `<form>` should not be here, but instead in the parent component, encompassing everything.